src/content/en/2021/cdn.md

```diff
@@ -205,20 +201,22 @@ Looking at third party domains below, you see a different trend in top CDN Provi
 
 For example, a CDN built specifically to deliver advertisements will be optimized for:
 
-1. High input-output (I/O) operations,
-2. Effective management of [long tail](https://en.wikipedia.org/wiki/Long_tail) content, and
-3. Geographical closeness to businesses requiring their services.
+- High input-output (I/O) operations
+- Effective management of [long tail](https://en.wikipedia.org/wiki/Long_tail) content
+- Geographical closeness to businesses requiring their services
+
+{# TODO aren't those the benefits of all CDNs? #}
```

Comment:
@Navaneeth-akam does this need clarification? I was a bit confused.

Reply:
Long tail content management is not something generic CDN's are good at by default. A purpose built CDN can be designed to manage this a bit better. As for geographical closeness, generic CDN will be deployed based on certain assessment but their geographical closeness is relative to Origin. Purpose built CDN will have a focused end user set where they will need deployment of server, which differs a bit from generic CDN

Reply:
The long tails makes sense (missed that to be honest), and guess that similarly applies to high input/output. I guess for geographical closeness I considered all CDNs to have good global presence but that’s maybe not the case?

Anyway let’s remove the TODO if happy with the wording.

```suggestion
```